docker/dev/config_docker.json

```diff
@@ -24,7 +24,8 @@
         "StorageDirectory": "/mattermost/data/",
         "AllowedLoginAttempts": 10,
         "DisableEmailSignUp": false,
-        "EnableOAuthServiceProvider": false
+        "EnableOAuthServiceProvider": false,
+        "AllowIncomingWebhooks": false
```

Comment:
That's the plan but I was going to wait until Asaad's UI changes before turning it on by default